utils/config.ts

```diff
@@ -16,15 +16,25 @@ import { NonEmptyString } from "@pagopa/ts-commons/lib/strings";
 // global app configuration
 export type IConfig = t.TypeOf<typeof IConfig>;
 // eslint-disable-next-line @typescript-eslint/ban-types
-export const IConfig = t.interface({
-  AzureWebJobsStorage: NonEmptyString,
+export const IConfig = t.intersection([
+  t.interface({
+    AzureWebJobsStorage: NonEmptyString,
 
-  COSMOSDB_KEY: NonEmptyString,
-  COSMOSDB_NAME: NonEmptyString,
-  COSMOSDB_URI: NonEmptyString,
+    COSMOSDB_KEY: NonEmptyString,
+    COSMOSDB_NAME: NonEmptyString,
+    COSMOSDB_URI: NonEmptyString,
 
-  isProduction: t.boolean
-});
+    // JWT Support
+    ISSUER: NonEmptyString,
+    PRIMARY_PRIVATE_KEY: NonEmptyString,
+    PRIMARY_PUBLIC_KEY: NonEmptyString,
+
+    isProduction: t.boolean
+  }),
+  t.partial({
+    SECONDARY_PUBLIC_KEY: NonEmptyString
+  })
```

Comment:
Can we extract the intersection of the JWT Configuration type ?

Reply:
Done in [[#IOCIT-301] refactor IConfig + TTL from config](https://github.com/pagopa/io-functions-lollipop/pull/14/commits/431284c74584bf8c56a106fcbc8cef8d057e3cf4) along with the move of the JWT_TTL on config (with fallback to 900 sec)